tests/overflow.rs

```diff
@@ -0,0 +1,42 @@
+extern crate gdk_pixbuf;
+use gdk_pixbuf::*;
+use std::mem::size_of;
+
+#[test]
```

Comment:
Why not `#[should_panic]` and `#[cfg(target_pointer_width = "64")]` for the tests? That way you test they actually panic.

Alternatively you can also catch the panic.